src/test/command_line_tests/mina_cli.ml

```diff
@@ -34,9 +34,10 @@ module MinaCli = struct
     Process.run ~prog:t.mina_exe ~args:[ "advanced"; "print-signature-kind" ] ()
 
   let advanced_compile_time_constants t ~config_file =
-    Process.run ~prog:t.mina_exe
-      ~args:
-        [ "advanced"; "compile-time-constants"; "--config-file"; config_file ]
+    Process.run
+      ~env:(`Extend [ ("MINA_CONFIG_FILE", config_file) ])
```

Comment:
MINA_CONFIG_FILE can now be set to specify the config file path for testing.

Reply:
This is unchanged from `compatible`, except that https://github.com/MinaProtocol/mina/pull/16167 set it up so that this variable leaked into all other CLI commands too.